kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/data/vector/DefaultConstantVector.java

```diff
@@ -120,4 +123,16 @@ public Row getStruct(int rowId) {
     public ArrayValue getArray(int rowId) {
         return (ArrayValue) value;
     }
+
+    @Override
```

Comment:
Seems incomplete to ignore DefaultConstantVector but this should never be used. I think possible options

1) Restrict DefaultConstantVector to only allow StructType for null values (for `NonExistentColumnConverter`)
2) Provide this implementation and add a test somewhere (not sure where to put this?)

Reply:
Option 1 + restrict it for Map and Array types for non-null values only?